benchmarks/benchmarks/bench_random.py

```diff
@@ -34,11 +34,11 @@ class Randint(Benchmark):
 
     def time_randint_fast(self):
         """Compare to uint32 below"""
-        np.random.randint(0, 2**30, size=10**5)
+        np.random.randint(low=0, high=2**30, size=10**5)
```

Comment:
IMHO unit tests should always use the most up-to-date versions of the API for clarity and consistency.  That is why I put one test in this PR for backwards compatibility in which I call the function using the "old" way.